src/dataLoaderService.ts

```diff
@@ -125,4 +126,23 @@ export class DataLoaderService {
       },
     });
   }
+
+  get userBalance() {
+    return this.getLoader<
+      {
+        userId: string;
+      },
+      GetBalanceResult
+    >({
+      type: 'userBalance',
+      loadFn: async ({ userId }) => {
+        return getBalance({
```

Comment:
Instead of "hacking" context, shouldn't we just provide `userId` as a parameter?

Reply:
It was due to auth protection, but I do agree it can be simplified, since I use it like that all over the place, refactored to just `{ userId }` 98f6bdec62641a30fd029de516b8a8ea894b166d